gitlab/resource_gitlab_project_hook.go

```diff
@@ -89,7 +84,6 @@ func resourceGitlabProjectHookCreate(d *schema.ResourceData, meta interface{}) e
 		MergeRequestsEvents:   gitlab.Bool(d.Get("merge_requests_events").(bool)),
 		TagPushEvents:         gitlab.Bool(d.Get("tag_push_events").(bool)),
 		NoteEvents:            gitlab.Bool(d.Get("note_events").(bool)),
-		BuildEvents:           gitlab.Bool(d.Get("build_events").(bool)),
```

Comment:
These got renamed to JobEvents which go-gitlab does support; https://github.com/xanzy/go-gitlab/blob/master/projects.go#L590